File: 20140218/fizzbuzz.py
def fbstr(i):
    if i % 15 == 0:
        s = "FizzBuzz"
    elif i % 3 == 0:
        s = "Fizz"
    elif i % 5 == 0:
        s = "Buzz"
    else:
        s = str(i)
    return s

def fizzbuzz(array):
    for i in array:
        print(fbstr(i))

fizzbuzz(range(1,100))

File: 20140618/fizzbuzz.c
#include <stdio.h>

int
main (void)
{
  int i;
  for (i = 1; i <= 100; i++)
    {
      if (i % 15 == 0)
        printf ("FizzBuzz\n");
      if (i % 3 == 0)
        printf ("Fizz\n");
      if (i % 5 == 0)
        printf ("Buzz\n");
      else
        printf ("%d\n", i);
    }
  return 0;
}
